docs/troubleshooting.md

```diff
@@ -62,9 +62,9 @@ You may also find the Stack Overflow topics relevant:
 
 Someone else from the community may have already asked a similar question or may be able to help with your problem. The Kubernetes team will also monitor [posts tagged kubernetes](http://stackoverflow.com/questions/tagged/kubernetes). If there aren't any existing questions that help, please [ask a new one](http://stackoverflow.com/questions/ask?tags=kubernetes)!
 
-## IRC
+## <a name="slack"></a>Slack
 
-The Kubernetes team hangs out on IRC at [`#google-containers`](https://botbot.me/freenode/google-containers/) on freenode.  Feel free to come and ask any and all questions there.
+The Kubernetes team hangs out on Slack in the `#kubernetes-users` channel.  Slack requires registration, but the [Kubernetes](https://kubernetes.slack.com) team is open invitation to [join](http://slack.kubernetes.io).  Feel free to come and ask any and all questions there.
```

Comment:
It's not obvious which of these links the user should click on to join or participate. I suggest being more explicit, like register HERE and access the channels HERE.